ng_sniffer/Makefile

```diff
@@ -0,0 +1,77 @@
+# Set the name of your application:
+APPLICATION = ng_sniffer
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../RIOT
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+# define the board feature that are needed for this project
+FEATURES_REQUIRED = periph_spi periph_gpio
+
+ifneq (,$(filter iot-lab_M3,$(BOARD)))
+  DRIVER ?= ng_at86rf231
+  export ATRF_SPI ?= SPI_0
+  export ATRF_CS ?= GPIO_11
+  export ATRF_INT ?= GPIO_12
+  export ATRF_RESET ?= GPIO_13
+  export ATRF_SLEEP ?= GPIO_14
+endif
+ifneq (,$(filter samr21-xpro,$(BOARD)))
+  DRIVER ?= ng_at86rf233
+  export ATRF_SPI ?= SPI_0
+  export ATRF_CS ?= GPIO_4
+  export ATRF_INT ?= GPIO_5
+  export ATRF_RESET ?= GPIO_6
+  export ATRF_SLEEP ?= GPIO_7
+  export ATRF_SPI_SPEED ?= SPI_SPEED_1MHZ
+endif
```

Comment:
yap, but at the moment we do not have a concept for initilizing the network devices in a structured manner yet